test/test-operation.js

```diff
@@ -32,260 +32,638 @@ var helpers = require('./helpers');
 var Sway = helpers.getSway();
 
 describe('Operation', function () {
-  var swaggerApi;
+  context('swaggerApiRelativeRefs', function () {
```

Comment:
@whitlockjc Okay, I fixed the test in `test-issues.js` but I'm not sure I understand your question about `context`. `context` is just an alias of `describe` and I used it to group the two test groups for the "regular" `swaggerApi` and one using relative refs. Could you clarify what you meant?

<img width="440" alt="screen shot 2016-12-23 at 00 31 16" src="https://cloud.githubusercontent.com/assets/406937/21443636/37bd3b36-c8a7-11e6-9575-5f2a0e72bd8f.png">